TMS.Net07.Homework.Task3.Part2/FactorialRecursion.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework.Task3.Part2
+{
+    class FactorialRecursion
+    {
+        public static void MainRecFactorial()
+        {
+            Console.WriteLine("Enter the number: ");
+            var boolean = int.TryParse(Console.ReadLine(), out var number);
+            if (boolean == false)
```

Comment:
`-1` is also a number 😉 
We risque to get into an infinite loop, because `Factorial` method stops recursion only when number becomes `0`, but in case of initial `-1` it'll never happen